indica/models/plasma.py

```diff
@@ -470,20 +460,33 @@ def initialize_variables(self, tstart: float, tend: float, dt: float):
         )
 
     def assign_profiles(
-        self, profile: str = "electron_density", t: float = None, element: str = "ar"
```

Comment:
Can remove element from the function call

Reply:
I wouldn't do that because this way we retain the possibility to assign different shapes to the different impurities. But I just saw an error which I will fix once I've merged in and send it back to you for confirmation.